docs/content/management-decision-logs.md

```diff
@@ -4,10 +4,11 @@ kind: management
 weight: 3
 ---
 
-OPA can periodically report decision logs to remote HTTP servers. The decision
-logs contain events that describe policy queries. Each event includes the policy
-that was queried, the input to the query, bundle metadata, and other information
-that enables auditing and offline debugging of policy decisions.
+OPA can periodically report decision logs to remote HTTP servers, using custom
+plugins, or to the console output; or any combination thereof.
+The decision logs contain events that describe policy queries. Each event includes
+the policy that was queried, the input to the query, bundle metadata, and other
+information that enables auditing and offline debugging of policy decisions.
```

Comment:
@ashutosh-narkar is that what you had in mind? I haven't found any mention of the previous restrictions that would now be gone, so I figured to just put it here 🤔

Reply:
This look good. Thanks!